Userland/Libraries/LibWebView/ViewImplementation.cpp

```diff
@@ -432,6 +432,19 @@ ErrorOr<LexicalPath> ViewImplementation::take_dom_node_screenshot(i32 node_id)
     return save_screenshot(bitmap);
 }
 
+ErrorOr<LexicalPath> ViewImplementation::dump_gc_graph()
+{
+    auto gc_graph_json = client().dump_gc_graph();
+
+    LexicalPath path { Core::StandardPaths::tempfile_directory() };
+    path = path.append(TRY(Core::DateTime::now().to_string("gc-graph-%Y-%m-%d-%H-%M-%S.json"sv)));
```

Comment:
Does this action exist on Serenity? If so, I think this will fail due to veils, we don't unveil /tmp. Other actions dump to the Downloads folder, which we do unveil.